### PR TITLE
Enable contract compile optimization per default

### PR DESCRIPTION
**Pull request by weilbith**
Related to: trustlines-protocol/blockchain#318

### CODE REVIEW

File: deploy_tools/cli.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def validate_address(ctx, param, value):
 optimize_option = click.option(
     "--optimize",
```
Comment (berndbohmeier) [inline deploy_tools/cli.py]: Please have a look here and define an off switch

Comment (berndbohmeier) [reply deploy_tools/cli.py]: http://click.palletsprojects.com/en/5.x/options/#boolean-flags

Comment (weilbith) [reply deploy_tools/cli.py]: Thanks for this good hint. Should have thought more when doing this.
:heavy_check_mark:

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,6 +1,11 @@
 ==========
 Change Log
 ==========
+`0.5.0`_ (2019-08-15)
+-------------------------------
+* Change ``--otimizate`` option's default to ``True``
```
Comment (berndbohmeier) [inline CHANGELOG.rst]: type: optimize

Comment (weilbith) [reply CHANGELOG.rst]: :heavy_check_mark:

**Pull request merged**